### ISSUE TITLE
[Keyboard Navigation-Home Page-Navigation]Focus order is not logical on the videos showing under Get inspired section.

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2597]
**User Experience:**
If focus moves here and there and does not follow proper order then it would be difficult to access any Page for a Screen reader dependent users.

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Repro Steps:-**
1. Navigate to url : https://makecode.com/
2. Navigate through all controls (say computing education, about Microsoft makecode, get inspired, resources elements etc)lying on the page.
3. Verify that focus order on the ‘Home Page' Page on Get inspired section.

**Actual Result:-**
Focus order is not logical on the videos showing under Get inspired section as focus move to some hidden element. Extra tabs are encountered while navigating the page using keyboard Tab key.

**Expected Result:-**
Focus order should be logical on the videos showing under Get inspired section as focus should not move to any hidden element.

**MAS Reference -**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

**Suggested Fix:**
tab index property is applicable to all visual elements on screen.  
<a href="/articles/" tab index="2">Articles</a> 
<a href="/resources/" tab index="3">Resources</a> 
 
Tips : 
'tab index' should only be used in cases where: 
•The default tab order is not ideal, AND 
•The tab order cannot be changed by rearranging items in the content and/or by altering the style sheet to reflect the best visual arrangement. 
 
**Key References for Fix:** 
http://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20160317/G59 
https://www.w3.org/WAI/GL/wiki/Creating_Logical_Tab_Order_with_the_Tabindex_Attribute 

**Please refer the Screen shot:**
[MAS2.4.3_Home Page_Get Inspired section.zip](https://github.com/Microsoft/pxt/files/1189817/MAS2.4.3_Home.Page_Get.Inspired.section.zip)

**Comment by veler** [comment]
@microsoftsam Need investigation, maybe linked to YouTube and can't fix it.

**Comment by veler** [comment]
This issue is linked to Youtube embed links. Out of our control.

**Comment by veler** [comment]
Needs to be forwarded to YouTube.

**Comment by abchatra** [comment]
External youtube bug closing
